keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/ArtifactVersion.kt

```diff
@@ -0,0 +1,64 @@
+package com.netflix.spinnaker.keel.api.artifacts
+
+import java.time.Instant
+
+/**
+ * An immutable data class that represents a published software artifact in the Spinnaker ecosystem.
+ *
+ * This class mirrors [com.netflix.spinnaker.kork.artifacts.model.Artifact], but without all the Jackson baggage.
+ * One notable difference from the kork counterpart is that this class enforces non-nullability of a few
+ * key fields without which it doesn't make sense for an artifact to exist in Managed Delivery terms.
+ */
+data class ArtifactVersion(
```

Comment:
This is just the renamed `PublishedArtifact` class with a few additions. Not sure why GitHub didn't detect that this time around.